cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1220,11 +1256,24 @@ installCommand = CommandUI {
      ++ " that, `configure` must be used.)\n"
      ++ " In contrast, without a sandbox, the flags to `install` are saved and"
      ++ " affect future commands such as `build` and `repl`. See the help for"
-     ++ " `configure` for a list of commands being affected.\n",
+     ++ " `configure` for a list of commands being affected.\n"
+     ++ "\n"
+     ++ "Installed executables will by default (and without a sandbox)"
+     ++ " be put into `~/.cabal/bin/`."
```

Comment:
@lspitzner, perhaps you could just drop the reference to `~/.cabal/bin` in the interest of getting this merged? Including the executable path in the help message could be added as future work.